.github/test-categories.yaml

```diff
@@ -32,6 +32,7 @@ main:
   - main-xml-jaxb
 database:
   - couchdb
+  - debezium-mysql
```

Comment:
We could perhaps combine these under a single debezium itests module. Otherwise we have to do the native compilation step on every debezium itest we add.